ramls/loan.json

```diff
@@ -153,6 +153,18 @@
             }
           }
         },
+        "enumeration": {
```

Comment:
Please, update API version.

Reply:
@bohdan-suprun do you mean "circulation" version in "ModuleDescriptior-template.json"-file?

Reply:
I mean circulation RAML and module-descriptor.

Reply:
Ok, thank you for the answer. Done.